package.json

```diff
@@ -32,7 +32,7 @@
     "lint": "npm-run-all lint:*",
     "prettier": "prettier --write ./**/*.{js,ts}",
     "security-checks": "git secrets --scan && git log -p | scanrepo",
-    "clean": "rimraf ./.build ./.artifact ./.serverless ./.webpack ./*.zip",
+    "clean": "rimraf ./.build ./.artifact ./.serverless ./*.zip",
```

Comment:
just to check if this will effect anything platform related, I think web pack is important for them

Reply:
No we don't need webpack. It was removed previously, but this one slipped past me somehow!